diktat-rules/src/test/resources/test/smoke/src/main/kotlin/Example6Expected.kt

```diff
@@ -18,3 +18,10 @@ val text =
        x 
     """
 
+val dockerFileAsText =
```

Comment:
not enough tests, you have created more conditions than in this testcase

Reply:
Added to fix tests